content/applications/inventory_and_mrp/inventory/shipping/setup/delivery_method.rst

```diff
@@ -1,100 +1,211 @@
 
-Delivery methods allow you to manage the transport company, the price
-and the destination. You can even integrate Odoo with external shippers
-to compute the real price and the packagings.
+.. seealso::
+   - :ref:`Third-party shipping carrier setup <inventory/shipping/third_party>`
+   - `Odoo Tutorials: Delivery Prices
+     <https://www.odoo.com/slides/slide/delivery-prices-613?fullscreen=1>`_
 
 Configuration
 =============
 
-Install the inventory module
-----------------------------
+To calculate shipping on sales orders and e-commerce, the *Delivery costs* module must be installed.
+To do so, navigate to the :menuselection:`Apps` application from the main Odoo dashboard. Then,
+remove the :guilabel:`Apps` filter, and type in `Delivery Costs` in the search bar. After finding
+the :guilabel:`Delivery Costs` module, click :guilabel:`Activate` to install it.
 
-Delivery methods are handled by the **Delivery costs** module. Go to
-**Apps** and search for the module. You should remove the **Apps** filter in
-order to see it :
-
-.. image:: delivery_method/setup05.png
+.. image:: delivery_method/install-module.png
    :align: center
+   :alt: Install the *Delivery Costs* module.
+
+Add shipping method
+===================
+
+To configure delivery methods, go to :menuselection:`Inventory app --> Configuration --> Shipping
+Methods`.
 
 .. note::
-    If you want to integrate delivery methods in your e-commerce,
-    you'll have to install the **eCommerce Delivery** module.
+   If the :guilabel:`Shipping Methods` option is not available from the :guilabel:`Configuration`
+   drop-down menu, verify whether the feature is enabled by following these steps:
 
-Configure the delivery method
------------------------------
+   #. Go to :menuselection:`Inventory app --> Configuration --> Settings`.
+   #. Scroll to the :guilabel:`Shipping` section and enable the :guilabel:`Delivery Methods` feature
+      by checking the corresponding checkbox.
 
-To configure your delivery methods, go to the **Inventory** module,
-click on :menuselection:`Configuration --> Delivery Methods`.
+   .. image:: delivery_method/enable-delivery.png
+      :align: center
+      :alt: Enable the *Delivery Methods* feature by checking the box in Configuration > Settings.
 
-First set a name and a transporter company.
+On the :guilabel:`Shipping Methods` page, add a method by clicking :guilabel:`New`. Doing so opens
+a form to provide details about the shipping provider, including:
 
-.. image:: delivery_method/setup03.png
-   :align: center
+- :guilabel:`Shipping Method` (*Required field*): the name of the delivery method, e.g. `flat-rate
+  shipping`, `same day delivery`.
+- :guilabel:`Provider` (*Required field*): define the shipping method. Choose the delivery service,
+  like Fedex, if using a third-party carrier. Ensure the integration with the shipping carrier is
+  properly installed and select the provider from the drop-down menu.
```

Comment:
Do we have a doc about configuring shipper integrations? If so, it might be good to link it here.